map_frame: map
local_frame: local
body_frame: body
imu_topic: /imu/data_raw
rs_m1_topic: /os1_points

# 回环检测线程频率
loop_rate: 5.0
# 局部构图线程频率
local_rate: 60.0

lio_builder:
  # 小于该距离的点云从单帧中去掉
  blind: 0.6
  # 单帧点云中每隔多少个点取一个点
  filter_num: 2
  # 单帧点云中intensity在此区间则保留
  min_pt_intensity: 12
  max_pt_intensity: 255
  # 局部地图维护长度
  cube_len: 500.0
  # 单帧点云观测范围
  det_range: 100.0
  # 单帧点云降采样分辨率
  resolution: 0.2
  # 距离边界move_thresh * det_range时移动local_ikdtree
  move_thresh: 1.5
  # 初始化时是否对齐重力
  align_gravity: true
  imu_ext_rot: [1, 0, 0, 0, -1, 0, 0, 0, -1]
  imu_ext_pos: [0.065, 0.025, -0.18]

loop_closure:
  activate: true
  rad_thresh: 0.2
  dist_thresh: 1.0
  time_thresh: 30.0
  loop_pose_search_radius: 10.0
  loop_pose_index_thresh: 20
  submap_resolution: 0.25
  submap_search_num: 20
  loop_icp_thresh: 0.01